tasks/main.yml

```diff
@@ -10,15 +10,16 @@
     # see https://github.com/NVIDIA/nvidia-docker/issues/1511
     - name: Collect distribution detail
       set_fact:
-        distribution: "centos8"
-      when: ansible_os_family == 'RedHat' and ansible_distribution_major_version == '8'
+        distribution: "rhel9.0"
```

Comment:
isn't it 9.1?

Reply:
yes but that does not exist in nvidia repo :D